emulators.tex

```diff
@@ -8,14 +8,127 @@ \chapter{Emulators}
 
 Download the MEGA65 emulator source code from \url{https://github.com/lgblgblgb/xemu}.
 
-Download pre-compiled versions from \url{https://github.lgb.hu/xemu/}.
+Download pre-compiled versions from \url{https://github.lgb.hu/xemu/}. Installers are
+available for Apple Macintosh, Windows and Linux.
 
-A live ISO image containing the emulator, documentation, and other tools is available from \url{Forum64.de}
+A live ISO image containing the emulator, documentation, and other tools is available from
+Forum64.de
 at \url{https://www.forum64.de/index.php?thread/104698-xemu-live-system-iso-file/\&postID=1549927\#post1549936}.
+Should you choose to use it you may find instructions on page \pageref{sec:live-iso-image}.
 
 \section{Using The Xmega65 Emulator}
+\index{emulator}
+
+\subsection{The keyboard}
+\index{keyboard}
+
+When using a real Mega 65 a keyboard layout related to the old Commodore machines is
+encountered. How can this be adapted to a PC/Mac ? It's not that hard. Most of the keys
+for writing stay the same - given an American layout. So also special characters reached
+by \specialkey{Shift} and a number are same to a U.S. keyboard.
+
+The closer you get to \specialkey{Enter} or \widekey{Backspace} keys you will find
+a whole column of keys missing. Yes, even a C64-keyboard had this extra set of keys. For
+Xemu those are mainly operated with the six keys above the cursor keys. There you also
+find \specialkey{RUN STOP}, completely off in comparison to C64 emulator like VICE. The reason is, a Mega 65 even has an \specialkey{Esc} key. So some keys are approaching a
+more modern layout.
+
+\begin{center}
+  \includegraphics[width=\linewidth]{images/xemu-extended-keyboard.png}
+\end{center}
+
+As you might spot quickly with this setup \specialkey{RUN STOP} \widekey{RESTORE} could
+be done single-handed.
+
+Please be aware this image shows graphics symbols being reached in the orientation the
+\megasymbolkey and \specialkey{Shift} key are positioned on a real Mega 65. On a PC
+keyboard it is crossovered, unfortunately. So, while \megasymbolkey \megakey{S} gives
+you the upper right corner of a rectangle \specialkey{Shift} \megakey{S} gives you the
+heart symbol every new C64 owner must have tried first in the eightees.
+
+For the PC styled special character keys the lower symbol is reached without the use
+of modifier keys. The upper symbol shows what you get in conjunction with the
+\specialkey{Shift} key.
+
+Xemu lets you have two joysticks (one at a time) on the numeric keypad.
```

Comment:
I wonder if it's worth adding a sentence: "This feature requires a PC keyboard with a numeric keypad, and is not available otherwise."

Reply:
I think there are also Macs with numerical keypads around.

Reply:
I didn't mean it as a Mac-specific comment. Lots of keyboards lack 10-keys. I personally don't like 10-keys and always buy external keyboards without them. I miss out on the keyboard joystick. :)

Reply:
> Xemu lets you have two joysticks (one at a time) on the numeric keypad.

Isn't my sentence already quite obvious that if you don't have a numpad you won't have the keys to control the joystick? Remapping is given in another section with the example of freeing up [F9][F11] functions keys to get BASIC listing scrolling back. Maybe a later edit there could explain on how to get twisted [:] and [;] working. Much later a gamer related mapping could be added to no longer have cursor keys as they were. Be it as it may, we'll never meet everyones preferences.

Reply:
It's fine, not going to require it. Was just thinking out loud.